gemhardware/include/gem/hw/amc13/AMC13Manager.h

```diff
@@ -119,6 +120,8 @@ namespace gem {
             xdata::Boolean monBackPressure;
             xdata::Boolean enableLocalTTC;
 
+	    xdata::Boolean enableLEMO;
```

Comment:
Hi Jared, I have removed the pointer and I have rearranged the new
parameter for LEMO trigger to be in L1A bag. However I see the changes I've
done to connections file were not commited (i. e. in the repository the
cmsgemos <https://github.com/jotadram6/cmsgemos>/setup
<https://github.com/jotadram6/cmsgemos/tree/release-v2/setup>/etc
<https://github.com/jotadram6/cmsgemos/tree/release-v2/setup/etc>/
addresstables
<https://github.com/jotadram6/cmsgemos/tree/release-v2/setup/etc/addresstables>
/connections_ch.xml is unchanged)

On Wed, Jun 29, 2016 at 5:07 PM, Jared Sturdy <notifications@github.com>
wrote:

> In gemhardware/include/gem/hw/amc13/AMC13Manager.h
> <https://github.com/cms-gem-daq-project/cmsgemos/pull/5#discussion_r68963588>
> :
>
> > @@ -119,6 +120,8 @@ namespace gem {
> >              xdata::Boolean monBackPressure;
> >              xdata::Boolean enableLocalTTC;
> >
> > +	    xdata::Boolean enableLEMO;
>
> Add this parameter into the localTriggerConfig bag (and the corresponding
> parameter to the L1AInfo class)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cms-gem-daq-project/cmsgemos/pull/5/files/ed6b55e649e562280a5028304c289fa6a86ef788#r68963588>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AApYfLLsDEJz5_GIcEWwcVmRlpMIK-hFks5qQooigaJpZM4JAR0Q>
> .
>



-- 
_______________________________
José David Ruiz Álvarez
CMS Group - Universidad de los Andes
CERN - Office: 42 - R - 036
_______________________________